staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -146,10 +146,13 @@ func (cnc *CloudNodeController) Run(stopCh <-chan struct{}) {
 
 // UpdateNodeStatus updates the node status, such as node addresses
 func (cnc *CloudNodeController) UpdateNodeStatus(ctx context.Context) {
-	instances, ok := cnc.cloud.Instances()
+	var instances cloudprovider.Instances
+	instancesV2, ok := cnc.cloud.InstancesV2()
 	if !ok {
-		utilruntime.HandleError(fmt.Errorf("failed to get instances from cloud provider"))
-		return
+		if instances, ok = cnc.cloud.Instances(); !ok {
+			utilruntime.HandleError(fmt.Errorf("failed to get instances from cloud provider"))
```

Comment:
If we are requiring cloud providers to implement the InstancesV2 method, then this change clearly requires a release note.

Reply:
We're not requiring it, but it should be in the release notes anyways.

Reply:
I believe having node controller fail if instanceV2 is not enabled(ok), is effectively requiring instanceV2.

Reply:
Updated release note.